src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -61,9 +62,11 @@
   // +----+-------------+------+-----------+-----------+-------+----+----------+----+-----------+------+-----------+-----------+----+----------+
   public static final String fetchAnswersAndCommentsQuery = "SELECT * FROM Answer LEFT JOIN " 
       + "(SELECT id, username FROM User) AnswerUsername ON Answer.author_id=AnswerUsername.id "
-      + "LEFT JOIN Comment ON Answer.id=Comment.answer_id LEFT JOIN "
-      + "(SELECT id, username FROM User) CommentUsername ON Comment.author_id=CommentUsername.id"
-      + " WHERE Answer.question_id=?;";
+      + "LEFT JOIN Comment ON Answer.id=Comment.answer_id "
+      + "LEFT JOIN (SELECT id, username FROM User) CommentUsername "
+      + "ON Comment.author_id=CommentUsername.id "
+      + "WHERE Answer.question_id=? "
+      + "ORDER BY Answer.date_time ASC;";
```

Comment:
I think `ASC` is unnecessary.

Reply:
but I think it is better since it is explicit on the order to someone who isn't so familiar with SQL